UI/src/view/LinearAssetLayer.js

```diff
@@ -171,16 +172,29 @@ window.LinearAssetLayer = function(params) {
   map.addLayer(indicatorLayer);
   indicatorLayer.setVisible(false);
 
+  var trafficSignReadOnlyLayer = new TrafficSignReadOnlyLayer({
+      layerName: layerName,
+      style: new PointAssetStyle('trafficSigns'),
+      collection: new TrafficSignsCollection(backend, 'trafficSigns', true),
```

Comment:
Cant' this `TrafficSignsCollection `initialized on application like `roadCollection`, just to not be necessary pass the `backend `to the layer